### PR TITLE
Permutation argument bug fixed

### LINKED ISSUE #63: Bug in permutation argument cycle

### PR DESCRIPTION
**Pull request by nkaskov**
Fixes #63.

### CODE REVIEW

File: include/nil/crypto3/zk/snark/systems/plonk/placeholder/prover.hpp [hunk 1]
```diff
@@ -29,6 +29,7 @@
 #define CRYPTO3_ZK_PLONK_PLACEHOLDER_PROVER_HPP
 
 #include <chrono>
+#include <cassert>
```
Comment (tshchelovek) [inline include/nil/crypto3/zk/snark/systems/plonk/placeholder/prover.hpp:32]: remove this include

File: test/systems/plonk/placeholder/placeholder.cpp [hunk 2]
```diff
@@ -339,6 +339,10 @@ BOOST_AUTO_TEST_CASE(placeholder_permutation_argument_test) {
 
     for (int i = 0; i < argument_size; i++) {
         BOOST_CHECK(prover_res.F[i].evaluate(y) == verifier_res[i]);
+        for (std::size_t j = 0; j < desc.rows_amount; j++) {
+            BOOST_CHECK(prover_res.F[i].evaluate(preprocessed_public_data.common_data.basic_domain->get_domain_element(
```
Comment (tshchelovek) [inline test/systems/plonk/placeholder/placeholder.cpp:343]: (maybe) add this check to other tests

**Review by tshchelovek — CHANGES_REQUESTED** [review]
minor changes needed, look at comments

**Review by SK0M0R0H — APPROVED** [review]

File: test/systems/plonk/placeholder/placeholder.cpp (hunk 2 above)
Comment (SK0M0R0H) [reply test/systems/plonk/placeholder/placeholder.cpp:343]: I agree that we need this check in other tests. 
However, it shouldn't be a part of permutation-related PR (imho)

**Review by tshchelovek — APPROVED** [review]

**Pull request merged**